portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -53,8 +53,9 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
       String username = (String) entity.getProperty("username");
       String comment = (String) entity.getProperty("comment");
+      int score = Integer.parseInt(entity.getProperty("score").toString());
```

Comment:
Is it possible to cast it ```(int) entity.getProperty("score")```? This seems kind of convoluted to cast it to string first.

Reply:
I think `int score = Math.toIntExact((Long)entity.getProperty("score"));` is probably the best solution. Casting directly to an int was giving me the error `java.lang.ClassCastException: java.lang.Long cannot be cast to java.lang.Integer`.

Reply:
Ah, I guess there's not a good way to do it. What you have here is fine. I also found

```((Integer) entity.getProperty("score")).intValue();```